docs/QuickstartDeployPortal.md

```diff
@@ -39,7 +39,7 @@ Below is a table that describes all of the parameters in the custom deployment t
 |Sql Admin Password|**Yes**|This is the admin password for your SQL Server. Note that there are some password requirements that you can read about [here](https://docs.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-ver15#password-complexity)|
 |Sql Location|**Yes**|This is set by default to deploy to the same region as the web application. If you set this to a different application, you will incure charges since your web application will be in one region and your server will be in another|
 |Fhir Version|**Yes**|Select the FHIR version (STU3, R4, or R5) that you want your FHIR Server to be on.|
-|Enable Export|No|Determines whether export will be enabled for this fhir instance. If true, a storage account will be created as part of the deployment. You will need owner or user-administrator permissions for this. If you set this to false, you can change it to true after deployment but you will have to create and link the storage account yourself.|
+|Enable Export|No|Determines whether export will be enabled for this fhir instance. If true, a storage account will be created as part of the deployment. If you set this to false, you can change it to true after deployment and add the storage account value. More info on the storage setting, check [Bulk Export](https://github.com/microsoft/fhir-server/blob/master/docs/BulkExport.md). Note that you will need owner or user-administrator permissions for this. If you encounter any storage permission or role assignment related issue, a quick workaround is that you set the option to false and ask your administrator to help update the setting later.|
```

Comment:
Just a slight rewording - 

If you set this to false, you can change it to true after deployment and add the storage account value. For more info on the storage setting, check [Bulk Export](https://github.com/microsoft/fhir-server/blob/master/docs/BulkExport.md). Note that you will need owner or user-administrator permissions for this. If you encounter any storage permission or role assignment related issues, a quick workaround is to set **Enable Export** to false and ask your administrator to help update the setting later.|

Reply:
I just updated it.

Reply:
Sounds good! you should be able to squash/merge but let me know if you can't.